scopes/lanes/diff/lane-diff-generator.ts

```diff
@@ -31,29 +40,49 @@ export class LaneDiffGenerator {
       throw new Error(`unable to run diff between "${fromLaneName}" and "${toLaneName}", they're the same lane`);
     }
     const legacyScope = this.scope.legacyScope;
-    this.fromLane = fromLaneName ? await legacyScope.lanes.loadLane(new LaneId({ name: fromLaneName })) : null;
-    const toLane = await legacyScope.lanes.loadLane(new LaneId({ name: toLaneName }));
-    if (!toLane) {
+    const fromLaneId = fromLaneName ? new LaneId({ name: fromLaneName }) : null;
+    const toLaneId = toLaneName ? new LaneId({ name: toLaneName }) : null;
+    const isFromOrToDefault = fromLaneId?.isDefault() || toLaneId?.isDefault();
+
+    if (!isFromOrToDefault && !toLaneId) {
       throw new Error(`unable to find a lane "${toLaneName}" in the scope`);
+    } else if (!isFromOrToDefault && toLaneId && fromLaneId) {
+      const toLane = await legacyScope.lanes.loadLane(toLaneId);
+      if (!toLane) throw new Error(`unable to find a lane "${toLaneName}" in the scope`);
+      const fromLane = fromLaneId ? await legacyScope.lanes.loadLane(fromLaneId) : null;
+      this.toLaneData = await this.mapToLaneData(toLane);
+      this.fromLaneData = fromLane ? await this.mapToLaneData(fromLane) : null;
+    } else if (fromLaneId?.isDefault() && !!toLaneId) {
```

Comment:
not sure if `!!` is needed. if it does, maybe `Boolean(toLaneId)` is more intuitive.